sdk/storage/storage-file/review/generated/models/index.ts

```diff
@@ -0,0 +1,3146 @@
+   * be an exact case-sensitive match with the origin that the user age sends to the service. You
+   * can also use the wildcard character '*' to allow all origin domains to make requests via CORS.
+   */
+  allowedOrigins: string;
+  /**
+   * The methods (HTTP request verbs) that the origin domain may use for a CORS request. (comma
+   * separated)
+   */
+  allowedMethods: string;
+  /**
+   * The request headers that the origin domain may specify on the CORS request.
+   */
+  allowedHeaders: string;
+  /**
+   * The response headers that may be sent in the response to the CORS request and exposed by the
+   * browser to the request issuer.
+   */
+  exposedHeaders: string;
+  /**
+   * The maximum amount time that a browser should cache the preflight OPTIONS request.
+   */
+  maxAgeInSeconds: number;
+}
+
+/**
+ * A listed directory item.
+ */
+export interface DirectoryItem {
+  name: string;
+}
+
+/**
+ * File properties.
+ */
+export interface FileProperty {
+  /**
+   * Content length of the file. This value may not be up-to-date since an SMB client may have
+   * modified the file locally. The value of Content-Length may not reflect that fact until the
+   * handle is closed or the op-lock is broken. To retrieve current property values, call Get File
+   * Properties.
+   */
+  contentLength: number;
+}
+
+/**
+ * A listed file item.
+ */
+export interface FileItem {
+  name: string;
+  properties: FileProperty;
+}
+
+/**
+ * Abstract for entries that can be listed from Directory.
+ */
+export interface FilesAndDirectoriesListSegment {
+  directoryItems: DirectoryItem[];
+  fileItems: FileItem[];
+}
+
+/**
+ * A listed Azure Storage handle item.
+ */
+export interface HandleItem {
+  /**
+   * XSMB service handle ID
+   */
+  handleId: string;
+  /**
+   * File or directory name including full path starting from share root
+   */
+  path: string;
+  /**
+   * FileId uniquely identifies the file or directory.
+   */
+  fileId: string;
+  /**
+   * ParentId uniquely identifies the parent directory of the object.
+   */
+  parentId?: string;
+  /**
+   * SMB session ID in context of which the file handle was opened
+   */
+  sessionId: string;
+  /**
+   * Client IP that opened the handle
+   */
+  clientIp: string;
+  /**
+   * Time when the session that previously opened the handle has last been reconnected. (UTC)
+   */
+  openTime: Date;
+  /**
+   * Time handle was last connected to (UTC)
+   */
+  lastReconnectTime?: Date;
+}
+
+/**
+ * An enumeration of directories and files.
+ */
+export interface ListFilesAndDirectoriesSegmentResponse {
+  serviceEndpoint: string;
+  shareName: string;
+  shareSnapshot?: string;
+  directoryPath: string;
+  prefix: string;
+  marker?: string;
+  maxResults?: number;
+  segment: FilesAndDirectoriesListSegment;
+  nextMarker: string;
+}
+
+/**
+ * An enumeration of handles.
+ */
+export interface ListHandlesResponse {
+  handleList?: HandleItem[];
+  nextMarker: string;
```

Comment:
`nextMarker` should be normalized to `continuationToken` across the board.

Reply:
Will do (in generated code too)

Reply:
#5553